MailCore/Models/Attachment.swift

```diff
@@ -53,22 +53,43 @@ public class Attachment: /* Hashable, */ EmbeddedObject, Codable, Identifiable {
 
     public var icon: MailResourcesImages {
         guard let uti else { return MailResourcesAsset.unknownFile }
+
+        if uti.conforms(to: .pdf) {
+            return MailResourcesAsset.pdfFile
+        }
+        if uti.conforms(to: .calendarEvent) || uti.conforms(to: .ics) {
```

Comment:
I think I prefer `if {} else if {} else {}` to `if {} if {} return`

If we miss a return statement, the compiler should warn us.

Here is what I did for the [share ext file types](https://github.com/Infomaniak/ios-core/blob/master/Sources/InfomaniakCore/ItemProviderRepresentation/NSItemProvider%2BDetect.swift) Let me know what you think.